packages/now-cli/src/util/dev/server.ts

```diff
@@ -800,6 +803,10 @@ export default class DevServer {
       this.output.success('Build completed');
     }
 
+    // Ensure that the dev cache directory exists so that runtimes
+    // don't need to create it themselves.
+    await fs.mkdirp(this.devCacheDir);
```

Comment:
I didn't do this originally because the builder/runtime needed to create a separate dir for each entrypoint anyway 🤷

Reply:
Doesn't hurt I think, and is useful to some runtimes that leverage this dir (for example: the tsconfig files in this PR put them directly in the cache dir, without subdirs, using a hashed filename).

Unfortunately the `recursive` argument to core `fs.mkdir()` wasn't introduced until v10.12.0, so I'm not sure if that's viable for widespread usage yet (unless we want to alter the `engines` field in CLI's package.json), so an external dep like `fs-extra` or `mkdirp` would still be needed.

Reply:
>  the tsconfig files in this PR put them directly in the cache dir, without subdirs

I'm not so sure that is correct though. Imagine other builders/runtimes that make the same assumption. We should prefix this directory with `now-node` to avoid collision.

Reply:
Added prefixing logic in 3b57dcfb9d7d9c598e53d51ba9c450eaae2e0b59.